data/free worlds start.txt

```diff
@@ -1418,6 +1574,56 @@ mission "FW Pirates 3.2B"
 				accept
 
 
+	
+mission "FW Pirates 3.2D"
+	name "Discretion is the Better Part"
+	description "Escape the pirate armada and report to Tomek."
+	priority
+	landing
+	to offer
+        has "FW Pirates 3.2C: active"
+	source Greenrock
+	destination Longjump
+	on fail
+        dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	on offer
+	set "fw pirates: victory"
+	conversation
+	    `The few surviving rebel ships immediately turn tail and flee the system. You can either follow them back to Free Worlds territory, or face the biggest fight of your life, alone.`
+
+
+
+mission "FW Pirates 3.1D"
+	name "Get the Freed Slaves to Safety"
+	description "Escort a fleet of freed slaves to Longjump and answer for your decision."
+	landing
+	to offer
+        has "FW Pirates 3.1C: active"
+	destination Longjump
+	on fail
+        dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	on offer
+	set "fw pirates: victory"
+	conversation
+	    	`You bring your ship to hover over the spaceport and announce that they have one hour to release their captives and leave the area on the ground. Cowed by your guns, the slave market complies. The few remaining pirates and local thugs throw their valuables into any trucks they can find and head for the hills.`
+		`	Some of the freed slaves are pilots, technicians, and engineers, and they commandeer a dozen assorted transports and freighters. You spend half a day helping them liberate slave pens and labor camps around the planet. Under the threat of your guns, the slaves help themselves to the property of their former masters, and the transports and freighters are crammed with enough wealth, equipment, and weapons to enable a fresh start on a new world. Slave-masters who don't comply lose their farms, factories, and in many cases, their lives.`
+		`	Then you turn your guns onto every piece of infrastructure you can find that supports the pirate fleet. After several hours, the pirate's shipyard, outfitter, and refueling stations are destroyed beyond any hope of repair. Tracing the ion trails of other ships that launched to attack you, you find and destroy their bases of operaion, too. Former slaves are quick to point out key infrastructure that would otherwise have escaped your notice. It will take Greenrock decades to rebuild their pirate fleet and support structure, and they'll have to rebuild it without slave labor.`
+		`	Now you must get this rag-tag fleet full of escaped slaves to safety and answer for your actions.`
+		launch
+	npc accompany save
+		government Merchant
+		personality escort timid
+		fleet
+			names "civilian"
+			variant
+				"Behemoth"
```

Comment:
Eesh, yeah, I forgot to come back and add in more ships. Thanks for pointing that out. I've turned it into 18 cargo and transport ships with thematic names.